src/main/java/com/kosa/mini/api/dto/member/MemberEditProfileDTO.java

```diff
@@ -31,8 +31,20 @@ public Member toEntity() {
                 .nickname(nickname)
                 .email(email)
                 .phoneNumber(phoneNumber)
-                .password(password)
                 .role(roleId)
                 .build();
     }
+
+    // Entity -> DTO 변환 메서드 (새로 추가)
+    public static MemberEditProfileDTO fromEntity(Member member) {
+        MemberEditProfileDTO dto = new MemberEditProfileDTO();
+        dto.setMemberId(member.getMemberId());
+        dto.setName(member.getName());
+        dto.setEmail(member.getEmail());
+        dto.setNickname(member.getNickname());
+        dto.setPhoneNumber(member.getPhoneNumber());
+        dto.setRoleId(member.getRole());
```

Comment:
```suggestion
        dto.setRoleId(member.getRole());
```

내 정보 수정에서 role 정보는 안불러와도 될 것 같아요~